examples/resource-class/rclass.yml

```diff
@@ -0,0 +1,14 @@
+--- 
+kind: ResourceClass
+metadata: 
```

Comment:
[root@dell-r620-01 kubernetes]# kubectl get resourceclass my.rc2 -o yaml
apiVersion: v1
kind: ResourceClass
metadata:
  creationTimestamp: 2017-05-30T05:27:48Z
  name: my.rc2
  resourceVersion: "15222"
  selfLink: /api/v1/namespaces/default/resourceclasses/my.rc2
  uid: b820fe43-44f8-11e7-94f7-bc305bf4e400
spec:
  resourceSelector:
  - matchExpressions:
    - key: Type
      operator: In
      values:
      - nvidia-gpu
status: {}